APCAppCore/APCAppCore/DataSubstrate/Model/APCSmartSurveyTask.m

```diff
@@ -571,13 +572,41 @@ - (ORKAnswerFormat*) rkChoiceAnswerFormat:(NSDictionary *)objectDictionary
         ORKTextChoice * choice = [ORKTextChoice choiceWithText:option.label detailText:detailText value:option.value exclusive:!localConstraints.allowMultipleValue];
         [options addObject: choice];
     }];
-    if (localConstraints.allowOtherValue) {
-        [options addObject:NSLocalizedStringWithDefaultValue(@"Other", @"APCAppCore", APCBundle(), @"Other", @"Spinner Option")];
+    if (localConstraints.allowOtherValue)
+    {
+        // Smart capitalization will lowercase the "Other" option if all the other answers are lowercase
+        if (sAPCSmartSurveyEnableOtherAutoCapitalization &&
+            [self allLowercaseOptions:localConstraints])
+        {
+            [options addObject:NSLocalizedStringWithDefaultValue(@"other", @"APCAppCore", APCBundle(), @"other", @"Spinner Option")];
```

Comment:
Rather than having two strings for translators to deal with, and since capitalization rules based on context are different in different languages, you should probably set it to the capitalized string and then if we're doing the smart auto cap thing, use lowercaseString to convert it, since that's what you're checking against for "all the other answers".